cmd/bad-key-revoker/main_test.go

```diff
@@ -445,3 +445,27 @@ func TestInvokeRevokerHasNoExtantCerts(t *testing.T) {
 	test.AssertEquals(t, len(mm.Messages), 1)
 	test.AssertEquals(t, mm.Messages[0].To, "b@example.com")
 }
+
+func TestBackoffPolicy(t *testing.T) {
+	backoff := &BackoffPolicy{Limit: time.Second * 60, Value: time.Second * 1, Minimum: time.Second * 1, Factor: 1.3}
```

Comment:
I think it would improve readability if we used newlines and indentation.
```suggestion
	backoff := &BackoffPolicy{
	Limit: time.Second * 60,
	Value: time.Second * 1,
	Minimum: time.Second * 1,
	Factor: 1.3
}
```